Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -15,9 +15,57 @@
 
 @implementation BPPacker
 
++ (NSMutableSet *)fullyExpandedTests:(NSArray *)configTestCases fromTestFile:(BPXCTestFile *)xctFile {
+    NSMutableSet *expandedTests = [NSMutableSet new];
+    
+    for (NSString *testCase in configTestCases) {
+        if ([testCase rangeOfString:@"/"].location == NSNotFound) {
+            [xctFile.allTestCases enumerateObjectsUsingBlock:^(NSString *  _Nonnull actualTestCase, NSUInteger idx, BOOL * _Nonnull stop) {
```

Comment:
Please remove the nullability annotations (_Nonnull) here. We don't have those annotations yet across the project. Adding this in this line would introduce warnings for the whole file, we'll address the nullability check later.